rasterio/rio/edit_info.py

```diff
@@ -58,7 +59,10 @@ def transform_handler(ctx, param, value):
         except ValueError:
             pass
         try:
-            retval = guard_transform(value)
+            if tastes_like_gdal(value):
```

Comment:
@sgillies Adding a comment as @brendan-ward suggested, but do we want to allow `$ rio edit-info --transform` to be a GDAL geotransform, or _only_ an affine transform?